### PR TITLE
Konvert ExtendElement

### PR DESCRIPTION
**Pull request by jrodbx**

### CODE REVIEW

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import com.squareup.wire.schema.internal.Util.appendIndented
+
+data class ExtendElement(
+  val location: Location,
+  val name: String? = null,
+  val documentation: String = "",
+  val fields: List<FieldElement> = emptyList()
+) {
+  fun toSchema(): String {
+    val builder = StringBuilder()
```
Comment (JakeWharton) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt]: other PRs use `buildString`

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    appendDocumentation(builder, documentation)
+    builder.append("extend ")
+        .append(name)
+        .append(" {")
+    if (!fields.isEmpty()) {
```
Comment (Egorand) [inline wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt]: nit: `fields.isNotEmpty()`

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt (hunk 1 above)
Comment (jrodbx) [reply wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt]: fixed

**Comment by jrodbx** [comment]
Also, reverted `ExtendElement#name`'s type back to `String` (auto conversion for some reason, chose `String?`)

File: wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt (hunk 2 above)
Comment (jrodbx) [reply wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ExtendElement.kt]: fixed

**Pull request merged**